x/ibc/02-client/keeper/keeper.go

```diff
@@ -168,11 +173,35 @@ func (k Keeper) SetCommitter(ctx sdk.Context, clientID string, height uint64, co
 	store.Set(types.KeyCommitter(clientID, height), bz)
 }
 
+// GetConsensusState introspects the past historical info at a given height and
+// returns the expected consensus state at that height.
+func (k Keeper) GetConsensusState(ctx sdk.Context, height int64) (exported.ConsensusState, bool) {
```

Comment:
@cwgoes  @mossid @AdityaSripal should this use `height` and `height-1` for `NexValidatorSet` and `ValidatorSet` as that's how it is defined in `x/ibc/02-client/client/utils/utils.go`?

I need help to figure out what it the correct logic

Reply:
I don't understand why we need `NextValidatorSet` here.